internal/fusefrontend_reverse/virtualfile.go

```diff
@@ -59,8 +73,10 @@ func (f *virtualFile) GetAttr(a *fuse.Attr) fuse.Status {
 	}
 	st.Ino = f.ino
 	st.Size = int64(len(f.content))
-	st.Mode = syscall.S_IFREG | 0400
+	st.Mode = virtualFileMode
 	st.Nlink = 1
 	a.FromStat(&st)
+	a.Owner.Uid = f.Uid
+	a.Owner.Gid = f.Gid
```

Comment:
This is not needed. The owner should already be set correctly from "Lstat(f.parentFile, &st)"